packages/mobile/.env.alfajoresstaging

```diff
@@ -0,0 +1,8 @@
+ENVIRONMENT=alfajoresstaging
+DEFAULT_TESTNET=alfajoresstaging
+BLOCKCHAIN_API_URL=https://alfajoresstaging-dot-celo-testnet.appspot.com/
+DEV_SETTINGS_ACTIVE_INITIALLY=true
+FIREBASE_ENABLED=true
+SECRETS_KEY=integration
```

Comment:
this should be production, but know that I see the shipped app to the play store uses integration as well. At some point we should change that.